docs/source/en/guides/collections.md

```diff
@@ -58,6 +58,45 @@ A `note` can also be attached to the item. This is useful to add additional info
 
 In addition to these base attributes, returned items can have additional attributes depending on their type: `author`, `private`, `lastModified`, `gated`, `title`, `likes`, `upvotes`, etc. None of these attributes are guaranteed to be returned.
 
+## List collections
+
+We can also get multiple collections with [`list_collection`]. It allows us to retrieve collections and filter them with certain parameters. Let's list all collections of the user `teknium`.
+```py
+>>> from huggingface_hub import list_collections
+
+>>> collections = list_collections(owner="teknium")
+```
+This returns an iterable of `Collection` objects. We can iterate over it to print number of upvotes for each collections.
+```py
+>>> for i in collections:
+...   print("Number of upvotes:", i.upvotes)
+Number of upvotes: 1
+Number of upvotes: 5
+```
+
+It is possible to do more advanced filtering. Let's get all model collections, sort by trending, and limit the count to 5.
```

Comment:
```suggestion
It is possible to do more advanced filtering. Let's get all collections containing the model [TheBloke/OpenHermes-2.5-Mistral-7B-GGUF](https://huggingface.co/TheBloke/OpenHermes-2.5-Mistral-7B-GGUF), sorted by trending, and limit the count to 5.
```

While testing the API I noticed that filtering by item allow to list collections containing a specific item, not only listing collections with "models". I think it's a good occasion to showcase a cool model added to a lot of collections (like [TheBloke/OpenHermes-2.5-Mistral-7B-GGUF](https://huggingface.co/TheBloke/OpenHermes-2.5-Mistral-7B-GGUF))